freak/post_processing_freak.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
```

Comment:
Can this script be integrated into the xml wrapper as a config file? Any reasons to keep separate?

Reply:
Mm, it probably could be. I wasn't actually very familiar with that until I saw the GSVA wrapper. I think it would need to be refactored and re-tested, primarily due to differences between cheetah and normal python. Your call if its worth the time to bring it in.